examples/scaling-criteo/03-Training-with-TF.ipynb

```diff
@@ -72,7 +72,7 @@
     "# External dependencies\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    import nvtabular as nvt
</pre><p>Do we want to update this to come from core as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I'm trying to get us set up to import from <code>merlin.transforms</code> instead of <code>nvtabular</code>, but I'm not sure it's going to fully land until after the upcoming release.   </p>